spec/hangman_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require './hangman'
+
+RSpec.describe Hangman do
```

Comment:
I imagine if you run rubocop on this you'll get some errors ;)

`rubocop -a spec/hangman_spec.rb` should fix most, if not all, of them.